### ISSUE TITLE
bubble up go-logging fixes

### ISSUE THREAD

**Issue by whyrusleeping** [ipfs/go-ipfs#3486]
Since https://github.com/whyrusleeping/go-logging/pull/1

will resolve #3383

**Comment by whyrusleeping** [comment]
Lets try to do this in 0.4.8

**Comment by whyrusleeping** [comment]
Has this been done? go-ipfs is using the latest version of go-log (1.2.0) but i'm not sure if that actually resolves this issue

**Comment by kevina** [comment]
It doesn't look like it.  I looked at the code currently used by go-log "gx/ipfs/QmQvJiADDe7JR4m968MwXobTCCzUqQkP87aRHe29MEBGHV" and it does not include the fix.

**Comment by sanderpick** [comment]
Looked at this earlier because I'm getting hit by the thread safety issue addressed by this fix: https://github.com/whyrusleeping/go-logging/commit/04b9b69f12907542d61926adbc4d70c3d48d0258

tl;dr spinning up multiple nodes in tests causes a concurrent map read. I updated `go-log` w/ latest `whyrusleeping/go-logging`, then realized `go-log` is scattered throughout sub-packages... any advice on how to automate something like that?

**Comment by sanderpick** [comment]
Interesting, bumped to go 1.11.2 from 1.10.4 and go-ipfs 0.4.18 and am no longer seeing the concurrent map read panic. ¯\_(ツ)_/¯.

Happy Thanksgiving y'all!

**Comment by Kubuxu** [comment]
Not relevant anymore.
